docs/cli/init.mdx

```diff
@@ -8,9 +8,42 @@ Places the Service Worker file into a given public directory of your application
 ## Usage
 
 ```bash
-$ npx msw init <PUBLIC_DIR>
+$ npx msw init <PUBLIC_DIR> [options]
 ```
 
+### Positionals
+
+#### `PUBLIC_DIR`
+
+A relative path to the public directory of your application.
```

Comment:
Should we link to the public directory docs here? https://mswjs.io/docs/getting-started/integrate/browser#where-is-my-public-directory

Reply:
Yes, there are already linked right below the options on this page.